packages/asset-service/src/generateAssetData/generateAssetData.ts

```diff
@@ -29,25 +32,35 @@ const generateAssetData = async () => {
             await getZapperTokens(),
             await getUnderlyingVaultTokens()
           ])
+
         const tokens = [
           ...ethTokens,
           ...yearnVaults,
           ...ironBankTokens,
           ...zapperTokens,
           ...underlyingTokens
         ]
-        const uniqueTokens = uniqBy(tokens, 'caip19') // Remove dups
+
+        let uniqueTokens = uniqBy(tokens, 'caip19') // Remove dups
+
+        // Remove blacklisted tokens
```

Comment:
good approach, but can you please do this outside the `if` block on line 26, such that this blacklist is not ethereum specific, and avoids the duplication on line 85?

somewhere on line 93 once we have the final data before we right it out would be a good place to do this filtering.

this will become a problem in the future with scam tokens on other chains, our asset ids are already chain-agnostic, and we want this solution to be chain agnostic.